lib/gemstash/cache.rb

```diff
@@ -11,6 +11,10 @@ def initialize(client)
       @client = client
     end
 
+    def flush
+      @client.flush
```

Comment:
Fixed in 195aa822fce8f413c9427f460f3aa4507a2f559c along with some other forwardable calls